src/style/color.rs

```diff
@@ -153,6 +153,43 @@ impl serde::Serialize for Color {
 
 #[cfg(feature = "serde")]
 impl<'de> serde::Deserialize<'de> for Color {
+    /// This is used to deserialize a value into Color via serde.
+    ///
+    /// This implementation uses the `FromStr` trait to deserialize strings, so named colours, RGB,
+    /// and indexed values are able to be deserialized. In addition, values that were produced by
+    /// the the older serialization implementation of Color are also able to be deserialized.
```

Comment:
Add some info about what the older serialization impl was. E.g. "Prior to 0.2x.y, Ratatui serialized Color as ..., but now it ... Both formats are able to be deserialized ..."